bananas_api/new_upload/validate.py

```diff
@@ -218,6 +224,10 @@ def validate_files(files):
         if obj:
             file_info["package_type"] = obj.package_type
             obj.file_info = file_info
+            if obj.package_type in CLASSIFIERS:
+                obj.classification = CLASSIFIERS[obj.package_type](obj)
+            else:
+                obj.classification = Classification.UNKNOWN
```

Comment:
This should be `None`. No classifier should be set yet for categories that don't support it yet.